content/release-10-14-0/whatsnew-10-14-0.md

```diff
@@ -41,8 +41,33 @@ Additionally, devices and assets can be searched globally (for reports or home d
 
 ### Advanced Software Management
 
-*tbd*
+##### Software
 
+There is a new way to manage software called Advanced Software Management. This feature provides better scaling and extends the software model with the `softwareType` property.
+
+For example, the following static SmartREST template sets 3 software templates:
+
+```csv
+140,nvm,0.39.1,apt,https://github.com/nvm-sh/nvm,containerd.io,1.6.0,apt,https://containerd.io/,nginx,1.21.6,container-image,https://hub.docker.com/_/nginx
+```
+
+This would result in the creation of the software artefacts below:
+
+![Advanced Software](/images/release-notes/advanced-software.png)
+
+In order to migrate to the new Software model devices should remove their `c8y_SoftwareList` fragment and set their software artefacts anew using the SmartREST template above.
```

Comment:
Is this described in the documentation how to migrate?

Reply:
Seems to be missing:
https://github.com/SoftwareAG/c8y-docs/pull/370

Reply:
We should still add a link here to the section where this will be explained.

Reply:
@MaciekSoftwareAG can you add that info to your PR?

Reply:
Oops, I merged it already. I'll create a new PR for that as soon as it is ready.